connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -399,17 +400,17 @@ object KustoDataSourceUtils {
     }
     try {
       writeModeParam = parameters.get(KustoSinkOptions.KUSTO_WRITE_MODE)
-      isTransactionalMode =
-        if (writeModeParam.isEmpty) true
-        else WriteMode.withName(writeModeParam.get) == WriteMode.Transactional
+      writeMode =
```

Comment:
We need to fail if some other things are configured
For example SparkIngestionProperties - should have a method - validateForStreaming - this will see if either csvMapping, tags or creationTime are set - and fail - as we dont support them in streaming

Reply:
Please correct me here @ohadbitt , while we do not support tags, we do support mapping don't we. Was just looking at the ingestion stat and saw the Mapping field. Hence was not sure. Or is it that we do not support mappingReference ?

```
{
    "Mapping": null,
    "Format": "Csv",
    "Compressed": true,
    "IngestionSource": "Stream"
}
```

Reply:
The other way - we dont support mapping and do support mappingReference 

Also need to disable the GenerateDynamicCsvMapping in streaming for the same reason

Reply:
Sure, added this test. This will fail in case csvMapping is passed and will pass when csvMappingReference is passed in